src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -8,17 +8,93 @@ import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Table, Pagination, TooltipWrapper, Picker } from 'data-transparency-ui';
 import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
-import { budgetColumns, budgetDropdownColumns, budgetDropdownFieldValues, totalBudgetaryResourcesColumn, apiSpendingTypes, budgetCategoriesCssMappingTypes, budgetCategoriesSort } from 'dataMapping/covid19/budgetCategories/BudgetCategoriesTableColumns';
-import { fetchDisasterSpending, fetchLoanSpending } from 'helpers/covid19/budgetCategoriesHelper';
+import { budgetColumns, budgetDropdownFieldValues, budgetCategoriesCssMappingTypes, budgetCategoriesSort, sortMapping } from 'dataMapping/covid19/budgetCategories/BudgetCategoriesTableColumns';
+import { fetchDisasterSpending, fetchLoanSpending } from 'helpers/disasterHelper';
 import ResultsTableLoadingMessage from 'components/search/table/ResultsTableLoadingMessage';
 import ResultsTableErrorMessage from 'components/search/table/ResultsTableErrorMessage';
 import BaseBudgetCategoryRow from 'models/covid19/budgetCategories/BaseBudgetCategoryRow';
 import { BudgetCategoriesInfo } from '../../../components/award/shared/InfoTooltipContent';
 
 
 const propTypes = {
-    type: PropTypes.string.isRequired,
-    subHeading: PropTypes.string
+    type: PropTypes.string.isRequired
+};
+
+
+const budgetDropdownColumns = {
+    total_spending: [
+        {
+            title: 'obligation',
```

Comment:
Is there a reason we're separating out the `Total Budgetary Resources` column instead of including it here?

Reply:
I believe so, because we don't always want to show the `Total Budgetary Resources` for every `total_spending` view. We don't want to show the `Total Budgetary Resources` column for `Object Classes`.

Reply:
That's right - thanks!